samples/client/petstore/ruby/lib/petstore/api/fake_api.rb

```diff
@@ -681,9 +681,9 @@ def test_enum_parameters_with_http_info(opts = {})
 
     # Fake endpoint to test group parameters (optional)
     # Fake endpoint to test group parameters (optional)
-    # @param required_string_group Required String in group parameters
-    # @param required_boolean_group Required Boolean in group parameters
-    # @param required_int64_group Required Integer in group parameters
+    # @param required_string_group [Integer] Required String in group parameters
+    # @param required_boolean_group [BOOLEAN] Required Boolean in group parameters
```

Comment:
In any case, I will investigate this issue 🚶

Reply:
Good catch! Is it because boolean is missing in the type mapping (https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractRubyCodegen.java#L63) ?

Reply:
I think this is the cause.
https://github.com/OpenAPITools/openapi-generator/blob/master/modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/RubyClientCodegen.java#L106-L107
However, I do not know why it is purposely to be the upper case...

Reply:
You're right.

Me neither. I suggest we fix it instead of guessing the historical reasons behind.

Reply:
But I would still recommend adding "boolean" to typeMapping in the Ruby abstract codegen class.

Reply:
@wing328 
> I suggest we fix it instead of guessing the historical reasons behind.
> But I would still recommend adding "boolean" to typeMapping in the Ruby abstract codegen class.

I agree :+1: 
In any case, I think this is a separate topic from this PR, so I think that it may be merged.

Reply:
PR merged. Thanks for the review.